internal/client/graphql/errors.go

```diff
@@ -82,19 +82,20 @@ func convertUserErrors(uErrs interface{}) (result []query.UserError) {
 
 	userErrorType := reflect.TypeOf((*query.UserError)(nil)).Elem()
 
-	if typ := reflect.TypeOf(uErrs); typ.Kind() == reflect.Slice {
-		s := reflect.ValueOf(uErrs)
-		for i := 0; i < s.Len(); i++ {
-			v := s.Index(i)
-			if v.Type().Kind() != reflect.Ptr {
-				v = v.Addr()
-			}
-			if v.Type().Implements(userErrorType) {
-				result = append(result, v.Interface().(query.UserError))
-			}
-		}
-	} else {
+	if typ := reflect.TypeOf(uErrs); typ.Kind() != reflect.Slice {
```

Comment:
@sourishkrout, I'm wondering if this refactoring is part of the upgrade, and can be addressed in a separate PR?

Is it part of the go lint/format?

Reply:
Yes, lint/format after upgrade requires to turn if/else into early returns/breaks wherever possible. I think those changes are safe since we have broad test coverage for this code.